lua/lspconfig/async.lua

```diff
@@ -67,4 +67,45 @@ function M.reenter()
   end
 end
 
+--- @param cmd string|string[]
+--- @param on_success function(string[]?)
+--- @param on_fail? function(integer, string[]?)
+function M.run_job(cmd, on_success, on_fail)
```

Comment:
If this is actually needed it should be separate PR. Or update the existing `run_command`. This seems like a distraction from the main goal of this PR.

Reply:
I'll make a separate PR. Updating the existing run_command would probably be a breaking change of the API, that's why a made a separate function